pkg/contexts/ocm/download/handlers/ocirepo/registration.go

```diff
@@ -70,22 +57,17 @@ func (r *RegistrationHandler) RegisterByName(handler string, ctx download.Target
 	return true, nil
 }
 
-func (r *RegistrationHandler) GetHandlers(ctx cpi.Context) registrations.HandlerInfos {
-	return registrations.NewLeafHandlerInfo("uploading an OCI artifact to an OCI registry", `
-The <code>artifact</code> downloader is able to transfer OCI artifact-like resources
-into an OCI registry given by the combination of the download target and the
-registration config.
-
-If no config is given, the target must be an OCI reference with a potentially
-omitted repository. The repo part is derived from the reference hint provided
-by the resource's access specification.
-
-If the config is given, the target is used as repository name prefixed with an
-optional repository prefix given by the configuration.
-
+func (r *RegistrationHandler) GetHandlers(_ cpi.Context) registrations.HandlerInfos {
+	return registrations.NewLeafHandlerInfo("download OCI artifacts", `
+The <code>`+PATH+`</code> downloader is able to download OCI artifacts
+as artifact archive according to the OCI distribution spec.
 The following artifact media types are supported:
-`+listformat.FormatList("", supportedMimeTypes...)+`
+`+listformat.FormatList("", artdesc.ArchiveBlobTypes()...)+`
+By default, it is registered for these mimetypes.
+
 It accepts a config with the following fields:
-`+listformat.FormatMapElements("", AttributeDescription()),
+`+listformat.FormatMapElements("", ociuploadattr.AttributeDescription())+`
```

Comment:
I think, the old code was fine.

It should add, that if no mime types are configured the supported mimes types are used.